JS2-week2/app.js

```diff
@@ -0,0 +1,72 @@
+let questions = []; // Array of all questions
+const form = document.getElementById("quiz-form");
+const questionsWrapper = document.getElementById("questions-wrapper");
+let idCounter = 1; // Increamenting each question ID
+
+// this function handles adding each question after submitting
+function submitQuestion() {
+  const formElements = form.elements;
+  const formElementsArray = Array.from(formElements);
+  const answers = formElements["answers"].value.split(",");
+  let correctAnswer = false;
+
+  questions.push(
+    creatQuestionObject(
+      formElements["question"].value,
+      createAnswersArray(answers)
+    )
+  );
+
+  createQuestionElement(questions);
+
+  idCounter++;
+}
+
+// creating question object
+function creatQuestionObject(question, answers) {
+  return {
+    id: idCounter,
+    question,
+    answers,
+  };
+}
+
+// create answers array
+function createAnswersArray(answers) {
+  let answersArray = [];
+  answers.forEach((answer) => {
+    answersArray.push({
+      text: answer.replace(" ", ""),
+      isCorrect: false,
+    });
+  });
+  answersArray[
+    Math.floor(Math.random() * answersArray.length)
+  ].isCorrect = true;
+  return answersArray;
+}
+
+// creating question element
+function createQuestionElement(questions) {
```

Comment:
I think a better name for that funcion would be `renderQuestions` or `printQuestions` as what is does is pronting all the quations in the questions array. This name suggests that it just creates an instance of an element related to one question. A good funcion name should contain the information on what the funcion does specifically.